_includes/footer.html

```diff
@@ -1,9 +1,56 @@
 {% raw %}
-<section class="text-white" id="copyright">
+<section class="text-white" id="footer">
     <div class="container">
         <div class="row">
+            <div class="col-lg-3 col-md-6 mb-4 mb-md-0">
+                <h3>　Main</h3>
```

Comment:
extra space here

Reply:
and in the other <h3> tags too

Reply:
I know it's not good, but Main won't come right up the row.

Reply:
ah i see now, no problem